src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/MediaTransportControls_themeresources.xaml

```diff
@@ -846,6 +846,11 @@
                                                     <FontIcon Glyph="&#xED3D;" />
                                                 </AppBarButton.Icon>
                                             </AppBarButton>
+                                            <AppBarButton x:Name="RepeatVideoButton" Style="{StaticResource AppBarButtonStyle}" MediaTransportControlsHelper.DropoutOrder="10" Visibility="Collapsed">
+                                                <AppBarButton.Icon>
+                                                    <FontIcon Glyph="&#xE8EE;" />
+                                                </AppBarButton.Icon>
+                                            </AppBarButton>
```

Comment:
Is this change part of the playback rate modifications?

Reply:
No. I detected that i add in the V1, when i add on the Generic. And do not on the V2.
Should i remove from this PR now?

Reply:
Removed. I will add in a new one.